composer.json

```
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": ">=8.0",
-        "bref/bref": "^1.0",
+        "bref/bref": "1 - 2",
```

I would recommend using this syntax instead:

```suggestion
        "bref/bref": "^1.0 || ^2.0",
```